content/millennium/r4/bulk-data/group-export.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Group Export | R4 API
+---
+
+# Group Export
+
+<%= beta_tag %>
+
+* TOC
+{:toc}
+
+## Overview
+
+The Group Export operation is used to obtain FHIR resources for all members of a specified group of patients.
+
+An Export process uses the [FHIR Asynchronous Request Pattern](https://hl7.org/fhir/R4/async.html) to kick off the export job for a group of patients. Once the export job is kicked off, the status endpoint may be called to check the progress of the job. Once the job is complete, the status endpoint returns a list of files which may be downloaded using the file retrieval endpoint. After the appropriate ndjson files are downloaded, the delete endpoint should be called to clean up the no longer needed files.
+
+The Group Export operation exports resources required for [United States Core Data for Interoperability (USCDI)](https://www.healthit.gov/isa/united-states-core-data-interoperability-uscdi). For a full list of resources see the [US Core Profiles](http://hl7.org/fhir/us/core/STU4/#us-core-profiles). When the _type parameter is used it returns only the FHIR resources specified. The _type parameter should be used to filter the resources required for a specific use case. Limiting the number of requested resources in the _type parameter is recommended as it can decrease response times.
```

Comment:
for 'United States Core Data for Interoperability' can you add V1 at the end to clarify the bulk export operation supports USCDI V1 not V2 or V3 etc.

Reply:
Added here: https://github.com/cerner/fhir.cerner.com/pull/899/commits/b4d173a788a1abe02a6e1d5f2880bc13e6044dd8#diff-6963b85421ccad81bf2c26b848074843817dfa021595650e04f7ff89e1e85e31R18